File: frontend/src/components/Cart/CartInfo.js
import React from "react";

const CartInfo = ({name, price, quantity}) => {
  return (
    <>
      <hr />
      <p className="flex justify-between font-medium text-lg my-2">
        <span>{quantity} x {name} (₹ {price})</span> <span>₹ {quantity*price}</span>
      </p>
    </>
  );
};

export default CartInfo;

File: src/src/components/Cart/cart_info.py
import reflex as rx

def CartInfo(item) -> rx.Component:
  return rx.fragment(
    rx.divider(),
    rx.chakra.box(
      rx.text(f'{item["quantity"]} x {item["name"]} ₹ {item["price"]}'),
      rx.text(f'₹ {item["total"]}'),
      class_name="flex justify-between font-medium text-lg my-2"
    )
  )